contracts/pcv/compound/ERC20CompoundPCVDeposit.sol

```diff
@@ -2,12 +2,7 @@
 pragma solidity ^0.8.0;
 
 import "./CompoundPCVDepositBase.sol";
-
-interface CErc20 {
-    function mint(uint256 amount) external returns (uint256);
-
-    function underlying() external returns (address);
-}
+import "./CErc20.sol";
```

Comment:
should merge this interface change anyway even though this PR will move to https://github.com/fei-protocol/ERC4626